services/dataplatform/versions.go

```diff
@@ -13,7 +13,7 @@ import (
 func (c *Client) GetVersions(ctx context.Context) ([]string, *dataplatform.APIResponse, error) {
 	versions, apiResponse, err := c.sdkClient.DataPlatformMetaDataApi.VersionsGet(ctx).Execute()
 	apiResponse.LogInfo()
-	return versions, apiResponse, err
+	return *versions.Items, apiResponse, err
```

Comment:
Did `VersionsGet` change its return type? Seems it initially returned []string directly but now it places it in the `Items` field?
We might need to check err before dereferencing now.

Reply:
@cbeti-ionos yes, before it was `[]string`, now it is `VersionsGet200Response`.

Reply:
@cbeti-ionos done in cdd9b58070e612723abae0806410ff7d12151ea5